book/src/tutorial/building.md

```diff
@@ -2,9 +2,10 @@
 
 ## Toolchain
 
-In order to compile for UEFI, an appropriate target must be installed. The
-easiest way to set this up is using a [rustup toolchain file]. In the root of
-your repository, add `rust-toolchain.toml`:
+In order to compile for UEFI, an appropriate target must be installed. Unless
+your operating system provides up to date packages, the easiest way to set this
```

Comment:
I think this should be rephrased since the important point isn't really whether the packages are up-to-date, it's whether the UEFI targets are packaged at all. Maybe something like:

```suggestion
your operating system provides packages for the Rust UEFI targets, the easiest way to set this
```